src/Relay/Connection/ConnectionInterface.php

```diff
@@ -35,12 +35,14 @@ public function setPageInfo(PageInfoInterface $pageInfo): void;
     /**
      * Get the total count.
      *
-     * @return int
+     * @return int|object|null
      */
```

Comment:
Need to specify the exact returned object class or interface (promise in this case, I guess). And updating the description would be nice too.